dummyserver/certs/server.ipv6addr.crt

```diff
@@ -1,16 +0,0 @@
------BEGIN CERTIFICATE-----
```

Comment:
Because they were introduced in #761, which is being reverted here. We could leave the tests in place if you think that's sensible.